### DISCUSSION TITLE
M600 macro for Ender 3 KE

### DISCUSSION THREAD

**Discussion by ceedee666** [Guilouz/Creality-Helper-Script-Wiki#577] [General]
Hello everyone,

I'm faily new to 3d printing and just started playing with macros. The M600 here looks much better then the M600 macro build 
in to the Ender 3 KE. 

Is there any reason the M600 macro would not work with the Ender 3 KE?
Would the Ender 3V3 oder K1 version be a better starting point. As far as I can tell there are only minimal differences. 

Thanks foru your help!
Christian

**Comment by Guilouz** [comment]
You can test latest script version for M600 support for KE.

**Reply by ceedee666** [reply]
The current version of the macro does not work on the KE. The calculaten of `y_park` leeds to invalid values. I'll sent a pull request with values working on my KE.

**Comment by Guilouz** [comment]
Now added.
